frontend/src/app/api/SearchOpportunityAPI.ts

```diff
@@ -1,37 +1,48 @@
 import "server-only";
 
-import { environment } from "src/constants/environments";
+import BaseApi from "src/app/api/BaseApi";
 import { QueryParamData } from "src/services/search/searchfetcher/SearchFetcher";
 import {
   PaginationOrderBy,
   PaginationRequestBody,
-  PaginationSortDirection,
   SearchFetcherActionType,
   SearchFilterRequestBody,
   SearchRequestBody,
 } from "src/types/search/searchRequestTypes";
-
-import BaseApi from "./BaseApi";
+import { SearchAPIResponse } from "src/types/search/searchResponseTypes";
+
+const orderByFieldLookup = {
+  opportunityNumber: "opportunity_number",
+  opportunityTitle: "opportunity_title",
+  agency: "agency_code",
+  postedDate: "post_date",
+  closeDate: "close_date",
+};
+
+type FrontendFilterNames =
+  | "status"
+  | "fundingInstrument"
+  | "eligibility"
+  | "agency"
+  | "category";
+
+const filterNameMap = {
+  status: "opportunity_status",
+  fundingInstrument: "funding_instrument",
+  eligibility: "applicant_type",
+  agency: "agency",
+  category: "funding_category",
+} as const;
```

Comment:
these types are all to support the refactor of buildFilters, which is... admittedly a lot of scope creep for this ticket